components/aws/actions/cloudwatch-logs-put-log-event/cloudwatch-logs-put-log-event.mjs

```diff
@@ -0,0 +1,64 @@
+import common from "../../common/common-cloudwatch-logs.mjs";
+import { toSingleLineString } from "../../common/utils.mjs";
+
+export default {
+  ...common,
+  key: "aws-cloudwatch-logs-put-log-event",
+  name: "CloudWatch Logs - Put Log Event",
+  description: toSingleLineString(`
+    Uploads a log event to the specified log stream.
+    [See docs](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/client-cloudwatch-logs/classes/putlogeventscommand.html)
+  `),
+  version: "0.1.0",
```

Comment:
According to a diff with `master` branch you should increment version here and other components as well right?
```suggestion
  version: "0.1.1",
```
Action components like
```
dynamodb-create-table.mjs
dynamodb-execute-statement.mjs
dynamodb-get-item.mjs
dynamodb-put-item.mjs
dynamodb-query.mjs
dynamodb-scan.mjs
dynamodb-update-item.mjs
dynamodb-update-table.mjs
eventbridge-send-event.mjs
lambda-create-function.mjs
lambda-invoke-function.mjs
s3-stream-file.mjs
s3-upload-file.mjs
sns-send-message.mjs
sqs-send-message.mjs
```
abd source components like 
```
new-emails-sent-to-ses-catch-all-domain.mjs
new-records-returned-by-cloudwatch-logs-insights-query.mjs
```

Reply:
Right, it's that this PR is on top of #2302, and that one bumps the versions. I'm not sure if right before merging there will be a need to rev, I'm going to check that before merging